src/common/utils/index.js

```diff
@@ -6,6 +6,17 @@ export const sortByDate = (a, b) => {
   return dateB - dateA;
 };
 
+export function sortByVersion(a, b) {
```

Comment:
This is pretty specific to forge, I would change the function name

Reply:
It should work for any `.` separated version string but I renamed it nontheless